prow/lib.sh

```diff
@@ -138,9 +138,10 @@ function cleanup_kind_multicluster() {
 }
 
 function setup_kind_cluster() {
-  IMAGE="${1:-kindest/node:v1.17.0}"
-  NAME="${2:-istio-testing}"
-  CONFIG="${3:-}"
+  IP_FAMILY="${1:-ipv4}"
```

Comment:
struggling how to add this option, I came up with using it as first parameter as it can be set always and default value doesn't change.